### DISCUSSION TITLE
关于项目启动次数

### DISCUSSION THREAD

**Discussion by tasty007** [zhayujie/chatgpt-on-wechat#181] [General]
你好，大大，
问下如果我想接入多个微信机器人，
是需要怎么操作？
1-只启动1个服务，不同微信号各扫一次吗(这个方法好像不太行)
2-在同个服务器里多次使用启动命令，开启多个进程，然后每个进程扫码登录？
3-不同的服务器，每个服务器只起一个程序，分别用一个微信号扫码登录？
大大有什么比较好的解决方案吗？

**Comment by zhayujie** [comment]
选择2
